admin_guide/cluster_autoscaler.adoc

```diff
@@ -0,0 +1,527 @@
+[[admin-guide-cluster-autoscaler]]
+= Managing the Cluster Auto Scaler
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[managing-autoscaler-overview]]
+== Overview
+
+This topic describes setting up the auto-scaler on your
+{product-title} cluster. The auto-scaler's remit is to ensure your
+pods have somewhere to run, and to also ensure there are no unneeded
+nodes when demand drops.
+
+Note: this feature is currently in tech-preview and the this guide
+only applies to running the auto-scaler on AWS.
+
+=== Auto Scale Groups
+
+An Auto Scale Group (ASG) is a logical representation of a set of
+machines. An ASG is configured with a minimum number of instances to
+run, a maximum number of instances that can be run and also the
+desired number of instances to run. An ASG starts by launching enough
+instances to meet its desired capacity. ASGs may start with a minimum
+of zero instances.
```

Comment:
Can users take advantage of all the standard AWS features, or are there limits to the way they can set up their ASG?

Reply:
I think it is the other way round: ASG/LC imposes a set of parameters and constraints, but perhaps I'm missing something in your question.

Reply:
> read the AWS docs about them

I think this bit is key; we have to be wary of documenting something that is a) typically well known and b) documented in greater detail elsewhere.